src/clear_map.h

```diff
@@ -289,6 +290,7 @@ static inline void MakeField(TileIndex t, uint field_type, IndustryID industry)
 	SetClearGroundDensity(t, CLEAR_FIELDS, 3);
 	SB(_me[t].m6, 2, 4, 0);
 	_me[t].m7 = 0;
+	_me[t].m8 = 0;
```

Comment:
This line is not necessary, but the previous line is also not necessary.
To keep it all the same way, better add it.